testing/minikube/install-hub

```diff
@@ -43,11 +45,21 @@ jupyterhub = get_hub_chart_dependency()
 check_call(['helm', 'repo', 'add', 'jupyterhub', jupyterhub['repository']])
 check_call(['helm', 'repo', 'update', 'jupyterhub'])
 
+# Extract the JupyterHub config from the helm chart values.yaml
```

Comment:
Could we extend this comment to explain why we need to do this? It would help me (right now) because I can't quite work out why we need to extract it from the chart and then load it via an argument.

(I think it is because we don't actually install the BinderHub chart when we run binderhub on minikube in "dev mode")

Reply:
Done. As for _why_ it's deployed separately, I've no idea